GWXtreme/eos_model_selection.py

```diff
@@ -409,7 +408,7 @@ def getEoSInterpFromMRFile(self, MRFile):
         max_mass = np.max(masses)
         return [s, masses, Lambdas, max_mass]
 
-    def getEoSInterp_parametrized(self, params, N=100):
+    def getEoSInterp_parametrized(self, params, N=100,m_min=1.0):
```

Comment:
@shaonghosh  your original code also has m_min defined statically for non_parametrized eos's :  https://github.com/shaonghosh/GWXtreme/blob/8bc92f3bf03872c70617978acd3e4bdade67caea/GWXtreme/eos_model_selection.py#L299 . However for parametrized Eos Intrepolation methods, (which I am guessing some one else added?) m_min is not defined statically. Why is that? shouldn't all of these be consistent?

Reply:
You are right about that. I think its an relic from the time I was developing. I think everywhere we should be using the correct range of masses. However, I want to do this carefully testing if the old results do not change. So, this will require a little bit of work. Can you please create a new issue on this, so that I can address this later?

Reply:
Okay, In the meantime, I am keeping the static min_mass then. Wants you close that issue I can change mine too! Hopefully my results wont change because this either!